cypress/integration/examples/dummy.spec.js

```diff
@@ -1,8 +1,8 @@
 /// <reference types="cypress" />
 
-context('Home page', () => {
+context('Salaries page', () => {
```

Comment:
Should we add more tests for the different pages and/or more detailed tests for the `Add your salary` page?

Reply:
I just fixed the one it was added, but for sure we can/need include more.

Reply:
Ok, let's talk about our testing strategy in slack or in a call